projects/components/preview/bootstrap/preview-entity-facet/preview-entity-facet.component.html

```diff
@@ -3,12 +3,12 @@
         <span class="text-muted px-1 cursor-pointer" (click)="toggleEntity(value)">
             <i class="far" [ngClass]="{'fa-check-square': !entityHidden(value), 'fa-square': entityHidden(value)}"></i>
         </span>
-        <span class="mr-auto mx-1">{{ value.displayValue }}</span>
+        <span class="mr-auto mx-1">{{ value.displayValue | sqValue }}</span>
```

Comment:
Thank you for the PR! Was there a specific reason to use `sqValue`? For internationalization, I would rather go with `sqMessage`.

Reply:
I'm used to use `sqValue` to decode data coming from Sinequa, and `sqMessage` for pure frontend i18n messages... But I may be wrong, and I didn't test with `sqMessage`...

Reply:
I guess if `sqValue` takes care of i18n, that's all right. However, we could go one step further and provide `sqValue` with the column of the entity. If there are custom formatters, they will be applied (for example, amounts of money).

```
{{ value.displayValue | sqValue:column }}
```

and (in ngOnInit):

```
this.column = this.appService.getColumn(this.entity);
```

Reply:
Any update on your side? Can you implement my proposed improvement?

Reply:
Sorry, I switched on other projects and forget to answer.
I'll check this and come back to you tomorrow morning.

Reply:
Hi Eric ! I implemented and tested (with success) your proposal. :)

Reply:
Looking good! thanks for the contrib!